doc/classes/EditorSettings.xml

```diff
@@ -1395,7 +1395,7 @@
 			The script editor's documentation comment color. In GDScript, this is used for comments starting with [code]##[/code]. In C#, this is used for comments starting with [code]///[/code] or [code]/**[/code].
 		</member>
 		<member name="text_editor/theme/highlighting/engine_type_color" type="Color" setter="" getter="">
-			The script editor's engine type color ([Vector2], [Vector3], [Color], ...).
+			The script editor's engine type color ([Node], [Node2D], [Control], ...).
```

Comment:
```suggestion
			The script editor's engine type color ([Object], [Mesh], [Node], ...).
```
I'd use a few different kinds of types here, using only `Node` types might create confusion that only those are covered